addons/website/views/website_templates.xml

```diff
@@ -393,20 +393,26 @@
                         </div>
                         <!-- Social -->
                         <div class="col-lg-6 pb16">
-                            <div class="s_share text-left no_icon_color" data-snippet="s_share" data-name="Social Media">
-                                <h5 class="s_share_title d-none">Follow us</h5>
-                                <a href="/website/social/facebook" class="s_share_facebook" target="_blank">
+                            <div class="s_social_media text-left no_icon_color" data-snippet="s_social_media" data-name="Social Media" contenteditable="false">
+                                <h5 class="s_social_media_title d-none" contenteditable="true">Follow us</h5>
```

Comment:
Hm, this "behavior" was rewritten quite some time.
Spec wise, we need to:
- Make the title editable
- Make everything else not editable (the social media links eg), as we don't want people to be able to create links between or inside existing social media links, that would break the options.js code which parse those links (or would need some more code and classes to handle that, which we don't want)
- Not complicate the DOM structure, ideally keep it as such to avoid making the PR 1000 lines diff as we need to modified 20 of occurrences of this snippet

At the end, I got inspired by https://github.com/odoo/odoo/commit/30db617bc8ff7727f40d7ef58c6578e84a13f284
I am not sure if there is a better way, but I would be interested to know if there is

Reply:
> I am not sure if there is a better way, but I would be interested to know if there is

I guess I keep forgetting there is not :grimacing: